pkg/aggregate/iterator.go

```diff
@@ -267,15 +268,27 @@ func (as *AggregateSet) updateCell(aggr AggrType, cell int, val float64) {
 // return the value per aggregate or complex function
 func (as *AggregateSet) GetCellValue(aggr AggrType, cell int) (float64, bool) {
 
-	if cell > as.maxCell || cell >= as.length || !as.validCells[cell] { // TODO: should >Len return NaN or Zero ?
+	if !isValidCell(cell, as) {
+		return math.NaN(), false
+	}
+
+	dependsOnSum := aggr == aggrTypeStddev || aggr == aggrTypeStdvar
+	dependsOnCount := dependsOnSum || aggr == aggrTypeAvg
```

Comment:
Isn't Avg also depends on sum?

Reply:
yes it does. Why?

Reply:
Then you should probably add `|| aggr == aggrTypeAvg` to the `dependsOnSum` because otherwise you might want the avg aggregator but the Sum aggregator is faulted so you will have corruption or panic